src/applications/disability-benefits/disability-rating-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -117,73 +133,69 @@ export default class DisabilityRatingCalculator extends React.Component {
+                id="ratingLabel"
               >
-                <i className="fas fa-plus-circle vads-u-padding-right--0p5" />
-                Add rating
-              </button>
-            </div>
-            <div className="vads-l-col--8" />
-          </div>
-          <br />
-          <div className="vads-l-row">
-            <div>
-              <button
-                className="calculate-btn"
-                onClick={this.handleSubmit}
-                disabled={!canSubmit}
+                Disability rating
+              </div>
+              <div
+                className="vads-l-col--6 small-screen:vads-l-col--6"
+                id="descriptionLabel"
               >
-                Calculate
-              </button>
-            </div>
-            <div className="vads-u-margin-left--1">
-              <button
-                className="va-button-link clear-btn vads-u-margin-y--1p5"
-                onClick={this.clearAll}
-              >
-                Clear all
-              </button>
+                Optional description
+              </div>
             </div>
           </div>
         </div>
-
-        {this.state.showCombinedRating === true && (
-          <CalculatedDisabilityRating
-            ratings={ratings}
-            calculatedRating={calculatedRating}
+        {disabilities.map((disability, idx) => (
+          <RatingRow
+            disability={disability}
+            ref={this.setRef}
+            key={idx}
+            indx={idx}
+            ratingRef={this.ratingRef}
+            isDeletable={disabilities.length > 2}
+            showErrors={this.state.showCombinedRating}
+            updateDisability={this.handleDisabilityChange}
+            removeDisability={this.handleRemoveDisability}
           />
-        )}
+        ))}
+        <div className="vads-u-padding-x--4">
+          <div>
+            <button
+              className="va-button-link vads-u-text-align--left vads-u-margin-y--1p5"
+              type="button"
+              onClick={this.handleAddRating}
+            >
+              <i className="fas fa-plus-circle vads-u-padding-right--0p5" />
+              Add rating
+            </button>
+          </div>
+          <div>
+            <button
+              type="button"
+              className="usa-button vads-u-width--auto"
+              onClick={this.handleSubmit}
+            >
+              Calculate
+            </button>
+            <button
+              className="usa-button va-button-link vads-u-margin-y--1p5 vads-u-margin-left--2"
+              onClick={this.clearAll}
+            >
+              Clear all
+            </button>
+          </div>
+          {this.state.showCombinedRating === true && (
+            <CalculatedDisabilityRating
+              ratings={ratings}
+              calculatedRating={calculatedRating}
+            />
+          )}
+        </div>
```

Comment:
Hiding whitespace changes clears this up quite a bit 🙂 These are mostly just grid/markup changes.